tests/test_validation.py

```diff
@@ -106,6 +109,11 @@
         (IpAddress, ipv4, ipaddress.IPv4Address(ipv4), ipv6, {"protocol": 4}),
         (IpAddress, ipv6, ipaddress.IPv6Address(ipv6), ipv4, {"protocol": 6}),
         (IpAddress, "", "", "1.2.3", {"blank": True}),
+        (IpNetwork, ipv4_n, ipaddress.IPv4Network(ipv4_n), "1.2.3/24", {}),
+        (IpNetwork, ipv6_n, ipaddress.IPv6Network(ipv6_n), "1.2.3/64", {}),
+        (IpNetwork, ipv4_n, ipaddress.IPv4Network(ipv4_n), ipv6_n, {"protocol": 4}),
+        (IpNetwork, ipv6_n, ipaddress.IPv6Network(ipv6_n), ipv4_n, {"protocol": 6}),
+        # (IpNetwork, "", "", "10.10.0.0/24", {"blank": True}),
```

Comment:
This is testing that a blank value initialization works and should not be uncommented

Reply:
I have fixed this by changing the value_fail (4th param) to "10.10.0/24"